### PR TITLE
how s the design?

### PR DESCRIPTION
**Pull request by Shubhs96**

### CODE REVIEW

**Comment by sirjan13** [comment]
Hi @Shubhs96 plz attach a screenshot for the proposed changes

File: index.html [hunk 1]
```diff
@@ -1,34 +1,174 @@
...
+	  </div>
+	</div>
+	<div class="footer">
+		
+	</div>
+</div>
 
+	</div>
+</div>
+<script type="text/javascript">
```
Comment (sirjan13) [inline index.html]: Make a separate file for JavaScript

File: index.html [hunk 2]
```diff
@@ -1,34 +1,174 @@
...
-body {
-    margin: 150px 0px 0px 450px;
-}
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>My IDE</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
+    <style type="text/css">
```
Comment (sirjan13) [inline index.html]: Segregate CSS code

**Comment by sirjan13** [comment]
Great Work.Just be careful with the segregation of code and adding ScreenShots of any new design you add in future.Hope to see your contributions in Future.

**Comment by Shubhs96** [comment]
ok Thank u ....yes I will post the changes screenshot too from now on.

**Pull request merged**